src/types/model.ts

```diff
@@ -132,9 +132,12 @@ export class Registry {
     genesisBlockNumber: number
   ): Promise<Registry> {
     const db = storage.getDB();
-    const str = await db.get(
-      getNetworkStorageKey(CONDITIONAL_ORDER_REGISTRY_STORAGE_KEY, network)
-    );
+    const str = await db
+      .get(
+        getNetworkStorageKey(CONDITIONAL_ORDER_REGISTRY_STORAGE_KEY, network)
+      )
+      .then((str) => str)
+      .catch(() => undefined);
```

Comment:
Why do we hide ANY error here?

Reply:
At this point, the only condition that can be thrown by `get` is `LEVEL_NOT_FOUND`, when requesting a key that doesn't exist. This is the case on first run when populating from contract genesis. I see your point may be more about making this specific to the `LEVEL_NOT_FOUND` throw, and bubbling up all else, which is fair.

Reply:
Specifically handled not found (database initialisation), and bubble up the rest.